cluster/get-kube.sh

```diff
@@ -63,7 +63,49 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+KUBERNETES_RELEASE_URL_OVERRIDDEN=false
```

Comment:
I think a slightly shorter way to do all of this might be to reverse the order that `KUBERNETES_RELEASE_URL` and `KUBERNETES_CI_RELEASE_URL` are initialized, i.e. replace all of this with

```bash
KUBERNETES_CI_RELEASE_URL="${KUBERNETES_CI_RELEASE_URL:-${KUBERNETES_RELEASE_URL:-https://storage.googleapis.com/kubernetes-release-dev/ci}}"
KUBERNETES_RELEASE_URL="${KUBERNETES_RELEASE_URL:-https://storage.googleapis.com/kubernetes-release/release}"
```

so that if `KUBERNETES_CI_RELEASE_URL` is set, we use that; else if `KUBERNETES_RELEASE_URL` is set, we use that; otherwise use the default value. I think that's the intended logic here?

assuming we do that, an explanatory comment might be a good idea.